ptbr/tests/test_training_cli.py

```
@@ -0,0 +1,613 @@
+    ) -> None:
+        """An empty output folder should be accepted for a new training run."""
+        out = tmp_path / "output"
+        out.mkdir()
+        result = runner.invoke(app, [str(cfg_file), "--output-folder", str(out)])
+        assert result.exit_code == 0
+        mock_launch.assert_called_once()
+
+    @mock.patch("ptbr.training_cli._launch_training")
+    def test_output_folder_allows_validation_artifacts(
+        self,
+        mock_launch: mock.MagicMock,
```

Ruff (0.15.1)

[warning] Unused method argument: `mock_launch`

(ARG002)